conf/cmake/Modules/FindCLANG_FORMAT.cmake

```diff
@@ -0,0 +1,56 @@
+# Find clang-format
```

Comment:
This looks like a file from somewhere else. It might be nice to add a direct link so that we can easily update it later.